packages/gdrcopy.spec

```diff
@@ -61,12 +61,10 @@ if [ -e /usr/bin/systemctl ]; then                                      \
     /usr/bin/systemctl daemon-reload                                    \
 fi
 
-%global __requires_exclude ^libcuda\\.so.*$
```

Comment:
why drop this?

Reply:
I removed it because there is no dependency on libcuda any more (or there should not be any since I disabled the exes). But it's true that it doesn't hurt to keep it.

Reply:
Added gain this line.